### DISCUSSION TITLE
@mantine/drawers similar to @mantine/modals

### DISCUSSION THREAD

**Discussion by marek-hanzal** [mantinedev/mantine#4099] [Feature requests]
Hello!

I **love** this library and I want to ask, if it would be possible to create a package similar to `@mantine/modals` only for Drawers?

Usecase is simple: I've `Menu.Dropdown` in which I've an action which uses `<Modal>`, but when it's displayed, it's immediately closed. So I've to use `modals.open()` which is OK, but I'd like to use a Drawer instead which I cannot use as it has the same drawback: it closes immediately. I understand somehow why, so that's the reason why I'm asking about this potential package.

Thank you a lot for response!

❤️

**Comment by wowczarczyk** [comment]
Yes, my use case is exactly the same, that would be lovely!

**Comment by KurtGokhan** [comment]
Related to https://github.com/orgs/mantinedev/discussions/5869

What if there was a single package for all overlays, maybe called `@mantine/overlay-manager`? It could handle modals, drawers, notifications etc.

**Comment by moreorover** [comment]
I'd love to see this as well! Having a package similar to `@mantine/modals` but for Drawers would be super useful for cases where modals aren't ideal. Has there been any consideration for this?
